### ISSUE TITLE
Resize button on mac titlebar looks weird

### ISSUE THREAD

**Issue by bedeho** [gabrielbull/react-desktop#106]
As you can see here, the resize button looks distorted and broken.

![screen shot 2018-01-02 at 07 34 14](https://user-images.githubusercontent.com/437292/34476275-9d5a2764-ef8f-11e7-8cf8-bb445bcc6bed.png)

Importantly, this only happens when I view this on an external monitor attached to my macbook, when I view it on my macbook, it looks fine. Any idea about why this would happen?

**Comment by gabrielbull** [comment]
Is one retina and the other one not? Anyway, if you should take a look and submit a PR that would be appreciated!

**Comment by bedeho** [comment]
Yes, that is correct. I also tried looking at it on a separate computer with another screen, and it gives a similar bad quality result.

I have tried to look into the component, but it was very complicated to figure out.
